internal/service/sagemaker/sagemaker_test.go

```diff
@@ -72,6 +72,27 @@ func TestAccSageMaker_serial(t *testing.T) {
 			"posix":                                                   testAccDomain_posix,
 			"spaceStorageSettings":                                    testAccDomain_spaceStorageSettings,
 		},
+		"Model": {
```

Comment:
whats the reason for this? this is only needed when tests need to be run serially (some resource can only be created only a single time at once). and even if that is the case here all tests function need to be lower case.

in any case, i dont this is needed

Reply:
Agreed. Reverted/Removed considering model constraints for serial run. There is work required to rename all old/legacy test functions if we intend to add in future.